main/models.py

```diff
@@ -2,58 +2,16 @@
 Common model classes
 """
 from django.conf import settings
-from django.contrib.postgres.fields import JSONField
 from django.db import transaction
-from django.db.models import PROTECT, DateTimeField, ForeignKey, Manager, Model
-from django.db.models.query import QuerySet
-from django.utils import timezone
-
-
-class TimestampedModelQuerySet(QuerySet):
-    """
-    Subclassed QuerySet for TimestampedModelManager
-    """
-
-    def update(self, **kwargs):
-        """
-        Automatically update updated_on timestamp when .update(). This is because .update()
-        does not go through .save(), thus will not auto_now, because it happens on the
-        database level without loading objects into memory.
-        """
-        if "updated_on" not in kwargs:
-            kwargs["updated_on"] = timezone.now()
-        return super().update(**kwargs)
-
-
-class TimestampedModelManager(Manager):
-    """
-    Subclassed manager for TimestampedModel
-    """
-
-    def update(self, **kwargs):
-        """
-        Allows access to TimestampedModelQuerySet's update method on the manager
-        """
-        return self.get_queryset().update(**kwargs)
-
-    def get_queryset(self):
-        """
-        Returns custom queryset
-        """
-        return TimestampedModelQuerySet(self.model, using=self._db)
-
-
-class TimestampedModel(Model):
-    """
-    Base model for create/update timestamps
-    """
-
-    objects = TimestampedModelManager()
-    created_on = DateTimeField(auto_now_add=True)  # UTC
-    updated_on = DateTimeField(auto_now=True)  # UTC
-
-    class Meta:
-        abstract = True
+from django.db.models import (
+    PROTECT,
+    DateTimeField,
+    ForeignKey,
+    JSONField,
+    Manager,
+    Model,
+)
+from mitol.common.models import TimestampedModel
 
 
 class AuditModel(TimestampedModel):
```

Comment:
I know it's not part of this PR, but I think we should get rid of the `PROTECT` rule for the `acting_user` field in this model too (for all the same reasons I mentioned before)